packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Is this required ?

Reply:
No not required, added while development forgot to remove.. updated [here](1c002b1da07d995fc126d14539617437b6371dd7)